x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/indexlifecycle/ShrinkStep.java

```diff
@@ -38,27 +38,16 @@ String getShrunkIndexPrefix() {
     @Override
     public void performAction(IndexMetaData indexMetaData, ClusterState currentState, Listener listener) {
         // if operating on the shrunken index, do nothing
-
-        Long lifecycleDate = LifecycleSettings.LIFECYCLE_INDEX_CREATION_DATE_SETTING.get(indexMetaData.getSettings());
-        if (lifecycleDate == null) {
-            throw new IllegalStateException("source index[" + indexMetaData.getIndex().getName() +
```

Comment:
You removed this validation but didn't re-add it in the copy state step that was added, do we want to remove this entirely? I don't think we want to